sdk/cosmos/azure_data_cosmos/Cargo.toml

```diff
@@ -28,6 +28,7 @@ tokio.workspace = true
 serde_json.workspace = true
 azure_identity.workspace = true
 clap.workspace = true
+time.workspace = true
```

Comment:
Just a note that this is a dev-dependency (hard to tell from the diff). It's only used by a test in `azure_data_cosmos` (to create an `AccessToken` response for testing our custom authorization policy)